linux57-tkg/linux57-tkg-config/prepare

```diff
@@ -1,68 +1,229 @@
 #!/bin/bash
 
-_tkg_srcprep() {
-  msg2 "Setting version..."
-  scripts/setlocalversion --save-scmversion
-  echo "-$pkgrel-tkg-${_cpusched}" > localversion.10-pkgrel
-  echo "" > localversion.20-pkgname
-
-  # add upstream patch
-  patch -p1 -i ../patch-"${pkgver}"
-
-  # ARCH Patches
-  if [ "${_configfile}" == "config_hardened.x86_64" ] && [ "${_cpusched}" == "cfs" ]; then
-    msg2 "Using linux hardened patchset"
-    patch -Np1 -i ../0012-linux-hardened.patch
+_define_vars() {
+ 
+  _basever=57
+  _basever_date="31/05/2020"
```

Comment:
I will never update this. Is this actually required for anything?

Reply:
I just forgot them them there x) They are not used anywhere.

Reply:
Good :+1:

Reply:
Fixed.